src/app/child-dev-project/attendance/attendance-details/attendance-details.component.html

```diff
@@ -47,6 +47,8 @@ <h1 mat-dialog-title>
         type="number"
         [value]="entity.countEventsPresent(forChild)"
         readonly
+        title="Number of days the child was present"
```

Comment:
these changes seem to be unrelated to what we are working on in this PR? If so, can you please revert them?

(to check out the version of one file from the master branch, you can for example use `git checkout master -- FILE_NAME`)